huaweicloud/services/lts/resource_huaweicloud_lts_keywords_alarm_rule.go

```diff
@@ -0,0 +1,725 @@
+			},
+			"user_name": {
+				Type:        schema.TypeString,
+				Required:    true,
+				ForceNew:    true,
+				Description: `Specifies the user name.`,
+			},
+			"topics": {
+				Type:        schema.TypeList,
+				Elem:        keywordsAlarmRuleNotificationRuleTopicSchema(),
+				Required:    true,
+				ForceNew:    true,
+				Description: `Specifies the topics.`,
+			},
+			"timezone": {
+				Type:        schema.TypeString,
+				Optional:    true,
+				Computed:    true,
+				ForceNew:    true,
+				Description: `Specifies the timezone.`,
+			},
+			"language": {
+				Type:        schema.TypeString,
+				Optional:    true,
+				ForceNew:    true,
+				Computed:    true,
+				Description: `Specifies the notification language.`,
+			},
+		},
+	}
+	return &sc
+}
+
+func keywordsAlarmRuleNotificationRuleTopicSchema() *schema.Resource {
+	sc := schema.Resource{
+		Schema: map[string]*schema.Schema{
+			"name": {
+				Type:        schema.TypeString,
+				Required:    true,
+				ForceNew:    true,
+				Description: `Specifies the topic name.`,
+			},
+			"topic_urn": {
+				Type:        schema.TypeString,
+				Required:    true,
+				ForceNew:    true,
+				Description: `Specifies the topic URN.`,
+			},
+			"display_name": {
+				Type:        schema.TypeString,
+				Optional:    true,
+				ForceNew:    true,
+				Computed:    true,
+				Description: `Specifies the display name.`,
+			},
+			"push_policy": {
+				Type:        schema.TypeString,
+				Optional:    true,
+				ForceNew:    true,
+				Computed:    true,
+				Description: `Specifies the push policy.`,
+			},
+		},
+	}
+	return &sc
+}
+
+func resourceKeywordsAlarmRuleCreate(ctx context.Context, d *schema.ResourceData, meta interface{}) diag.Diagnostics {
+	cfg := meta.(*config.Config)
+	region := cfg.GetRegion(d)
+
+	// createKeywordsAlarmRule: Create an LTS KeywordsAlarmRule.
+	var (
+		createKeywordsAlarmRuleHttpUrl = "v2/{project_id}/lts/alarms/keywords-alarm-rule"
+		createKeywordsAlarmRuleProduct = "lts"
+	)
+	createKeywordsAlarmRuleClient, err := cfg.NewServiceClient(createKeywordsAlarmRuleProduct, region)
+	if err != nil {
+		return diag.Errorf("error creating LTS client: %s", err)
+	}
+
+	createKeywordsAlarmRulePath := createKeywordsAlarmRuleClient.Endpoint + createKeywordsAlarmRuleHttpUrl
+	createKeywordsAlarmRulePath = strings.ReplaceAll(createKeywordsAlarmRulePath, "{project_id}", createKeywordsAlarmRuleClient.ProjectID)
+
+	createKeywordsAlarmRuleOpt := golangsdk.RequestOpts{
+		KeepResponseBody: true,
+		OkCodes: []int{
+			200,
+		},
+		MoreHeaders: map[string]string{"Content-Type": "application/json"},
+	}
+
+	createKeywordsAlarmRuleOpt.JSONBody = utils.RemoveNil(buildCreateKeywordsAlarmRuleBodyParams(d, cfg))
+	createKeywordsAlarmRuleResp, err := createKeywordsAlarmRuleClient.Request("POST", createKeywordsAlarmRulePath, &createKeywordsAlarmRuleOpt)
+	if err != nil {
+		return diag.Errorf("error creating KeywordsAlarmRule: %s", err)
```

Comment:
```suggestion
		return diag.Errorf("error creating Keywords alarm rule: %s", err)
```

please replace them all

Reply:
fixed